pkg/catalog/model.go

```diff
@@ -56,6 +58,38 @@ type Tag struct {
 	CommitID string
 }
 
+// AddressType is the type of an entry address
+type AddressType int32
+
+const (
+	// AddressTypeUnknown indicates that the address might be relative or full.
+	// Used only for backward compatibility and should not be used for creating entries.
+	AddressTypeUnknown AddressType = 0
+
+	// AddressTypeRelative indicates that the address is relative to the storage namespace.
+	// For example: "/foo/bar"
```

Comment:
I don't understand.  `namespace_test.go` has _no leading slashes_ on its relative addresses; doesn't this refer to the same thing?

Reply:
Done.